### ISSUE TITLE
Sporadic Crash After request to `/api/v1/channels/<id>/create`

### ISSUE THREAD

**Issue by blaskovicz** [mattermost/platform#1051]
We are in the process of trialing a v1.0.0 server. I have downloaded it from the requested releases url and run with `./bin/platform` after configuring a postgres database.

After startup (and sometimes during usage) the server crashes:

```
$ ./bin/platform

[10/14/15 12:43:06] [INFO] Current version is 1.0.0 (2467/Fri Oct  2 16:47:14 UTC 2015/a049b34dfb6b7c6251bcc0642284b6f4a0bff8ec)
[10/14/15 12:43:06] [INFO] Current working directory is /home/<user>/mattermost/mattermost
[10/14/15 12:43:06] [INFO] Loaded config file from  /home/<user>/mattermost/mattermost/config/config.json
[10/14/15 12:43:06] [INFO] Server is initializing...
[10/14/15 12:43:06] [INFO] Pinging sql master database
[10/14/15 12:43:06] [INFO] Pinging sql replica-0 database
[10/14/15 12:43:06] [DEBG] Initializing user api routes
[10/14/15 12:43:06] [DEBG] Initializing team api routes
[10/14/15 12:43:06] [DEBG] Initializing channel api routes
[10/14/15 12:43:06] [DEBG] Initializing post api routes
[10/14/15 12:43:06] [DEBG] Initializing web socket api routes
[10/14/15 12:43:06] [DEBG] Initializing file api routes
[10/14/15 12:43:06] [DEBG] Initializing command api routes
[10/14/15 12:43:06] [DEBG] Initializing admin api routes
[10/14/15 12:43:06] [DEBG] Initializing oauth api routes
[10/14/15 12:43:06] [DEBG] Initializing webhook api routes
[10/14/15 12:43:06] [DEBG] Parsing server templates at 
 /home/<user>/mattermost/mattermost/api/templates/
[10/14/15 12:43:06] [DEBG] Initializing web routes
[10/14/15 12:43:06] [DEBG] Using static directory at  /home/<user>/mattermost/mattermost/web/static/
[10/14/15 12:43:06] [DEBG] Parsing templates at  /home/<user>/mattermost/mattermost/web/templates/
[10/14/15 12:43:06] [INFO] Starting Server...
[10/14/15 12:43:06] [INFO] Server is listening on :8065
[10/14/15 12:43:06] [INFO] RateLimiter is enabled
[10/14/15 12:43:06] [DEBG] /api/v1/websocket
[10/14/15 12:43:06] [DEBG] /api/v1/websocket
[10/14/15 12:43:06] [DEBG] /api/v1/users/status
[10/14/15 12:43:06] [DEBG] /api/v1/websocket
[10/14/15 12:43:07] [DEBG] /api/v1/websocket
[10/14/15 12:43:07] [DEBG] /api/v1/users/status
[10/14/15 12:43:07] [DEBG] /api/v1/websocket
[10/14/15 12:43:07] [DEBG] /api/v1/websocket
[10/14/15 12:43:07] [DEBG] /api/v1/websocket
[10/14/15 12:43:07] [DEBG] /api/v1/websocket
[10/14/15 12:43:07] [DEBG] /api/v1/websocket
[10/14/15 12:43:07] [DEBG] /api/v1/websocket
[10/14/15 12:43:07] [DEBG] /api/v1/websocket
[10/14/15 12:43:08] [DEBG] /api/v1/websocket
[10/14/15 12:43:08] [DEBG] /api/v1/websocket
[10/14/15 12:43:10] [DEBG] /api/v1/users/status
[10/14/15 12:43:10] [DEBG] /api/v1/channels/5kakdn4pajg3zqngwwggh9i94e/create

panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x5034f2]

goroutine 177 [running]:
github.com/mattermost/platform/api.func·010()
        /home/travis/gopath/src/github.com/mattermost/platform/api/post.go:192 +0x41e2
created by github.com/mattermost/platform/api.fireAndForgetNotifications
        /home/travis/gopath/src/github.com/mattermost/platform/api/post.go:420 +0x124

goroutine 1 [chan receive]:
main.main()
        /home/travis/gopath/src/github.com/mattermost/platform/mattermost.go:68 +0x6d1
```

**Comment by coreyhulen** [comment]
Thanks for the report.  I'm 99% sure this is the same issue as https://mattermost.atlassian.net/browse/PLT-414

**Comment by blaskovicz** [comment]
Hey @coreyhulen. I tried viewing that issue after signing up on the Atlassian site but it says "permission violation."

**Comment by it33** [comment]
@blaskovicz you should have permissions now, sorry about that. When core team creates tickets, they default to private until reviewed, triaged and turned public, once in a while we miss the public setting,

**Comment by crspeller** [comment]
Hopefully fixed with #1142
